runtime/near-vm-runner/Cargo.toml

```diff
@@ -44,7 +46,7 @@ base64 = "0.13"
 default = ["wasmer0_vm", "wasmtime_vm", "wasmer1_vm"]
 wasmer0_vm = [ "wasmer-runtime" ]
 wasmtime_vm = [ "wasmtime", "anyhow"]
-wasmer1_vm = [ "wasmer", "wasmer-types", "wasmer-compiler-singlepass" ]
+wasmer1_vm = [ "wasmer", "wasmer-types", "wasmer-compiler-singlepass",  "wasmer-compiler-cranelift", "wasmer-engine-native", ]
```

Comment:
This will enable & compile the cranelift dependency even if we don't use it. Would it be better to do something like

```
        #[cfg(feature = "wasmer_compiler_cranelift")]
        let engine = wasmer_engine_native::Native::new(wasmer_compiler_cranelift::Cranelift::default())
                .engine();
        #[cfg(feature = "wasmer_compiler_singlepass")]
        let engine = wasmer_engine_native::Native::new(wasmer_compiler_singlepass::Singlepass::default())
                .engine();
```

In this case, compile-time features allow us to avoid compiling dependency altogether, so the pay off is relatively high.

Reply:
That's true, however experiments with various combinations of backend/serialization formats are easier the current way.